src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,7 +1,14 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validator.Email;
+
 public class UserRequestDto {
+    @Email
     private String email;
```

Comment:
Think of the fields as some characteristics of the object, and should they be always present for this object IRL or maybe optional? And make them null/not-null based on that thinking, so let's add not-null for email